### PR TITLE
Improvements for build and delivery

### PR DESCRIPTION
**Pull request by alevkin**
- Added build scripts for CentOS, Ubuntu
- Added push assets to GitHub Releases
- WIP for MacOS

### CODE REVIEW

**Comment by algys** [comment]
Please add minimal readme (how to build pkg, docker etc.)

**Review by algys — CHANGES_REQUESTED** [review]
need minimal readme/how-to/examples

**Comment by alevkin** [comment]
> Please add minimal readme (how to build pkg, docker etc.)

Done

**Review by algys — APPROVED** [review]

**Comment by algys** [comment]
build failed, so probably have some issues in CMakeLists.txt

**Review by algys — CHANGES_REQUESTED** [review]
fix build

**Review by algys — APPROVED** [review]

File: cicd/run [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    --rm --name=${_name}-${_dist}-build \
+    -ti ${_name}-${_dist}-build \
+    ${_command}
+}
+
+function sub_help(){
+  _subcommand=$1
+  case $_subcommand in
+    build)
+      echo "Usage: $ProgName build [options] {projact_name}"
```
Comment (Eenae) [inline cicd/run:40]: projact

**Pull request merged**